plugins/ips/ips.plugin.zsh

```diff
@@ -0,0 +1,3 @@
+# easy get ip - internal & external
+alias ipe="curl ipinfo.io/ip"
```

Comment:
```suggestion
alias ipe="curl -w '\n' ipinfo.io/ip"
```
[to remove](https://stackoverflow.com/questions/29497038/why-does-a-curl-request-return-a-percent-sign-with-every-request-in-zsh) the `%` at the end of the display :beers: